spec/xcpretty/parser_spec.rb

```diff
@@ -28,6 +28,11 @@ module XCPretty
       @parser.parse(SAMPLE_BUILD)
     end
 
+    it "parses analyze target" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.